src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -312,26 +315,37 @@ private CoverageProvider getCoverageProvider(Run<?, ?> build, FilePath workspace
         }
 
         Logger logger = new Logger(listener.getLogger());
+
         InstanceProvider<CoverageProvider> provider = new InstanceProvider<CoverageProvider>(
                 Jenkins.getInstance(),
                 COBERTURA_PLUGIN_NAME,
                 COBERTURA_CLASS_NAME,
                 logger
         );
-        CoverageProvider coverage = provider.getInstance();
+        CoverageProvider coberturaCoverage = provider.getInstance();
+
+        InstanceProvider<CoverageProvider> jacocoProvider = new InstanceProvider<CoverageProvider>(
+                Jenkins.getInstance(),
+                JACOCO_PLUGIN_NAME,
+                JACOCO_CLASS_NAME,
+                logger
+        );
+        CoverageProvider jacocoCoverage = jacocoProvider.getInstance();
 
-        if (coverage == null) {
+        if (coberturaCoverage == null && jacocoCoverage == null) {
             return null;
         }
 
+        CoverageProvider coverage = Objects.firstNonNull(coberturaCoverage, jacocoCoverage);
```

Comment:
Im not sure this logic is sound. Cobertura plugin could be available but only jacoco report could have been generated. We want to select the provider based on which one has coverage available.

Ideally we would support a hybrid coverage provider that can aggregate coverage results from all available providers. This can be a follow up/enhancement however